lib/processor.js

```diff
@@ -26,6 +27,52 @@ function getContentfulClient (accessToken, spaceId, host) {
 }
 
 /**
+ * Fetch common content for a certain space.
+ *
+ * @param {Object} options plugin config
+ * @param {Object} entries entries fetched from contentful
+ *
+ * @return {Object}        file mapping object with common content added
+ */
+function getCommonContentForSpace (options, entries) {
+  if (!options.common) {
+    return entries
+  }
+
+  const spaceId = options.space_id
+  const accessToken = options.access_token
+  const host = options.host
```

Comment:
Not a big deal, but...

Could we maybe create a helper function to help with `spaceId`, `accessToken` and `host`...
We've got 3 almost similar lines here as below. 

```
const spaceId = options.space_id
const accessToken = options.access_token
const host = options.host
```

Or we could move this logic into `getContentfulClient` and change the params?
I'm thinking of s.th. like

`getContentfulClient( options, file )`

which deals with the fallback logic?